packages/purpose-process/test/testClonePurpose.ts

```diff
@@ -0,0 +1,395 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  decodeProtobufPayload,
+  getMockAgreement,
+  getMockEService,
+  getMockPurpose,
+  getMockPurposeVersion,
+  getMockTenant,
+  readLastEventByStreamId,
+  writeInReadmodel,
+} from "pagopa-interop-commons-test/index.js";
+import {
+  Purpose,
+  PurposeClonedV2,
+  agreementState,
+  generateId,
+  purposeVersionState,
+  tenantKind,
+  toPurposeV2,
+  toReadModelAgreement,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
+import { formatDateAndTime, genericLogger } from "pagopa-interop-commons";
+import {
+  duplicatedPurposeTitle,
+  purposeCannotBeCloned,
+  purposeNotFound,
+  tenantKindNotFound,
+} from "../src/model/domain/errors.js";
+import { addOnePurpose } from "./utils.js";
+import {
+  agreements,
+  postgresDB,
+  purposeService,
+  purposes,
+  tenants,
+} from "./purposeService.integration.test.js";
+
+export const testClonePurpose = (): ReturnType<typeof describe> =>
+  describe("clonePurpose", async () => {
+    beforeAll(() => {
+      vi.useFakeTimers();
+      vi.setSystemTime(new Date());
+    });
+    afterAll(() => {
+      vi.useRealTimers();
+    });
+    it("should write on event-store for the cloning of a purpose", async () => {
+      vi.useFakeTimers();
+      vi.setSystemTime(new Date());
```

Comment:
There could be deleted since there are set in the `beforeAll` test hook

Reply:
Removed, thanks